src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -28,5 +29,11 @@ public enum Features : int
         /// Filter(A, A[@Value] = 2)
         /// </summary>
         DisableRowScopeDisambiguationSyntax = 0x4,
+
+        /// <summary>
+        /// Enable Identifier support for describing column names
+        /// Added on 20ieth July 2022.
+        /// </summary>
+        SupportIdentifiers = 0x8,
```

Comment:
>SupportIdentifiers [](http://example.com/codeflow?start=8&length=18)

nit: "identifier support" is too broad... 
Although "Identifier Args instead of Strings for field names" is too verbose.  Maybe something in the middle?